tailwind.config.js

```diff
@@ -41,7 +41,10 @@ module.exports = {
         lightgreen: 'var(--color-lightgreen)',
         oebblack: 'var(--color-black)',
         link: 'var(--color-link)',
-        oebgrey: 'var(--color-lightgray)'
+        oebgrey: 'var(--color-lightgray)',
+        grey85: 'var(--color-grey85)',
```

Comment:
To get different greys tailwind best practice is to do it like this:
![image](https://github.com/user-attachments/assets/001b35fa-aa75-47b3-92e8-615f2fbab73c)